pyverbs/libibverbs.pxd

```diff
@@ -4,7 +4,7 @@
 include 'libibverbs_enums.pxd'
 from libc.stdint cimport uint8_t, uint16_t, uint32_t, uint64_t
 from posix.time cimport timespec
-
+cimport pyverbs.libibverbs_enums as e
```

Comment:
no need, there's an "include 'libibverbs_enums.pxd'" line, so you can directly use the enums

Reply:
Thanks, didn't know that's how it works.